coderd/telemetry/telemetry.go

```diff
@@ -1235,6 +1232,7 @@ type NetworkEvent struct {
 	Time                time.Time               `json:"time"`
 	Application         string                  `json:"application"`
 	Status              string                  `json:"status"` // connected, disconnected
+	Version             string                  `json:"version"`
```

Comment:
```suggestion
	ClientVersion             string                  `json:"client_version"`
```

And move it below `ClientType`

Reply:
I had it as `Version` since we'll end up using these events for `coderd` and agents too, but for those we can just not set `ClientVersion` and add `CoderdVersion`.